sale_order_line_menu/views/sale_order_line_views.xml

```diff
@@ -0,0 +1,220 @@
+                    domain="[('state','=','cancel')]"
+                />
+                <separator />
+                <filter
+                    name="flt_invoice_status_no"
+                    string="Nothing to Invoice"
+                    domain="[('invoice_status','=','no')]"
+                />
+                <filter
+                    name="flt_invoice_status_upselling"
+                    string="Upselling Opportunity"
+                    domain="[('invoice_status','=','upselling')]"
+                />
+                <filter
+                    name="flt_invoice_status_to_invoice"
+                    string="To Invoice"
+                    domain="[('invoice_status','=','to invoice')]"
+                />
+                <filter
+                    name="flt_invoice_status_invoiced"
+                    string="Fully Invoiced"
+                    domain="[('invoice_status','=','invoiced')]"
+                />
+                <separator />
+                <filter name="flt_date_order" string="Order Date" date="date_order" />
+            </xpath>
+            <xpath expr="//group" position="inside">
+                <filter
+                    name="groupby_invoice_status"
+                    string="Invoice Status"
+                    context="{'group_by' : 'invoice_status'}"
+                />
+                <filter
+                    name="groupby_date_order"
+                    string="Order Date"
+                    context="{'group_by' : 'date_order'}"
+                />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="action_orders_lines" model="ir.actions.act_window">
+        <field name="name">Order Lines</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">sale.order.line</field>
+        <field name="view_mode">list,form,pivot</field>
+        <field
+            name="search_view_id"
+            ref="sale_order_line_menu.sale_order_line_search"
+        />
+        <field
+            name="domain"
+        >[('display_type', 'not in', ['line_section', 'line_note'])]</field>
+    </record>
+
+    <record
+        id="ir_action_order_line_list_window_view"
+        model="ir.actions.act_window.view"
+    >
+        <field eval="1" name="sequence" />
+        <field name="view_mode">list</field>
+        <field name="view_id" ref="sale_order_line_menu.sale_order_line_list" />
+        <field name="act_window_id" ref="sale_order_line_menu.action_orders_lines" />
+    </record>
+
+    <record
+        id="ir_action_order_line_form_window_view"
+        model="ir.actions.act_window.view"
+    >
+        <field eval="2" name="sequence" />
+        <field name="view_mode">form</field>
+        <field
+            name="view_id"
+            ref="sale_order_line_menu.sale_order_line_view_form_no_create"
+        />
+        <field name="act_window_id" ref="sale_order_line_menu.action_orders_lines" />
+    </record>
```

Comment:
Can't this be compressed into 1 record? (Or is it some weird new v18 style thing?)

Reply:
it's pre-commit that change style